hibernate-core/src/main/java/org/hibernate/tuple/entity/AbstractEntityTuplizer.java

```diff
@@ -40,12 +40,7 @@
 import org.hibernate.proxy.ProxyFactory;
 import org.hibernate.tuple.IdentifierProperty;
 import org.hibernate.tuple.Instantiator;
-import org.hibernate.type.AssociationType;
-import org.hibernate.type.BasicType;
-import org.hibernate.type.ComponentType;
-import org.hibernate.type.CompositeType;
-import org.hibernate.type.EntityType;
-import org.hibernate.type.Type;
+import org.hibernate.type.*;
```

Comment:
We try to avoid wildcard imports; could you please remove them?

[This IDE setup](https://github.com/hibernate/hibernate-ide-codestyles) might be useful.

Reply:
Sure, will do.

Reply:
I tried using the IDE setup files, but they might need updating for IntelliJ IDEA 2021.2.3.  Even after importing I had to change the thresholds manually for imports.  Nevertheless, I got the wildcards removed.